pkg/reconciler/v1alpha1/revision/resources/deploy.go

```diff
@@ -132,6 +132,7 @@ func makePodSpec(rev *v1alpha1.Revision, loggingConfig *logging.Config, observab
 	if rev.Status.ImageDigest != "" {
 		userContainer.Image = rev.Status.ImageDigest
 	}
+	userContainer.TerminationMessagePolicy = corev1.TerminationMessageFallbackToLogsOnError
```

Comment:
it occurs to me a user may set this, so perhaps we should add it to our ignored field list, or only set this if unspecified?

Reply:
Good one. 

https://github.com/knative/serving/blob/master/docs/spec/errors.md#container-image-fails-at-startup-on-revision

Makes it sound like the container will always be run that way, but it makes sense to only set this if unset from the outside IMHO.